docs/getting-started/kubernetes.md

```diff
@@ -6,52 +6,52 @@ sidebar_position: 1
 
 ## Prerequisites
 
-- This manual uses Helm to deploy Dragonfly on a Kuberenetes cluster. See  [Install Helm](https://helm.sh/docs/intro/install/)
+- This manual uses Helm to deploy Dragonfly on a Kuberenetes cluster. See [Install Helm](https://helm.sh/docs/intro/install/)
 - A Kuberenetes cluster (See [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/) or [Minikube](https://minikube.sigs.k8s.io/docs/start/) if you want to experiment locally)
-- Pick a version frome [here](https://github.com/dragonflydb/dragonfly/pkgs/container/dragonfly%2Fhelm%2Fdragonfly) e.g. `VERSION=v0.15.0`
+- Pick a version frome [here](https://github.com/dragonflydb/dragonfly/pkgs/container/dragonfly%2Fhelm%2Fdragonfly) e.g. `VERSION=$VERSION`
```

Comment:
Can you please modify to 
For latest version set `VERSION=v$VERSION`
Or pick a version from [here]

(Also note that you were missing the 'v')